Utilities/rANS/CMakeLists.txt

```diff
@@ -9,11 +9,17 @@
 # submit itself to any jurisdiction.
 
```

Comment:
Ok. Since the root dependency is there only to provide the root-persistency to Metadata, alternatively you can move the Metadata.h to DataFormats (perhaps directly to EncodedBlocks.h) and remove ROOT::Core from the rans CMakefile.